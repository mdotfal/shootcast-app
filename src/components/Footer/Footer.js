import React from 'react';

function Footer() {
  return (
    <div>
      <footer>ShootCast 2020</footer>
    </div>
  );
};

export default Footer;