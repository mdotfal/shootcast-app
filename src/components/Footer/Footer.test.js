import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe( `Footer component`, () => {

  it( `renders without crashing`, () => {
    const div = document.createElement( 'div' );
    ReactDOM.render( <Footer />, div );
    ReactDOM.unmountComponentAtNode( div );
  })

  it(Â `renders without errors`, () => {
    const wrapper = shallow( <Footer /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })
})