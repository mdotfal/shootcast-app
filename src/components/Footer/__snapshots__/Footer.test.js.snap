// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders without errors 1`] = `
<div>
  <footer>
    ShootCast 2020
  </footer>
</div>
`;
